src/views/PipelineView.vue

```diff
@@ -43,6 +40,14 @@ export default {
     path(route) {
       return '/' + this.pipelineInfo.name + '/' + route
     },
+    setSource(indx) {
```

Comment:
Not entirely sure what is going on so some documentation could help?

Reply:
👍

Reply:
I was think about the set source method. We can come back to it.